plugins/chat/spec/integration/post_chat_quote_spec.rb

```diff
@@ -211,4 +211,50 @@
   ensure
     InlineOneboxer.invalidate("https://en.wikipedia.org/wiki/Hyperlink")
   end
+
+  it "handles nested chat transcripts in posts" do
+    SiteSetting.external_system_avatars_enabled = false
+    channel = Fabricate(:chat_channel)
+    message1 = Fabricate(:chat_message, chat_channel: channel, user: post.user)
+    message2 = Fabricate(:chat_message, chat_channel: channel, user: post.user)
+    md = ChatTranscriptService.new(channel, message1.user, messages_or_ids: [message1.id]).generate_markdown
+    message1.update!(message: md)
```

Comment:
May be slightly confusing that message 1 content is replaced with a transcript of itself, but not awfully so. 🙂

Reply:
Oh you are right that is wrong, meant to use `message2` here :laughing: